### PR TITLE
[T3A3][W10-A1]Li Kai

### PR DESCRIPTION
**Pull request by li-kai**

### CODE REVIEW

**Comment by li-kai** [comment]
Ready for review.

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -25,31 +45,71 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
...
+    /**
+     * Takes in an address and processes it to return an array of address components.
+     * @param address		the address to process and split.
+     * @return				the array containing different parts of an address.
+     */
+    private static String[] splitAddressToComponents(String address) {
+    	final String[] addressComponents = address.split(ADDRESS_SPLIT_REGEX);
+        for (int i = 0; i < addressComponents.length; i++) {
+        	addressComponents[i] = addressComponents[i].trim();
+		}
```
Comment (OuyangDanwen) [inline src/seedu/addressbook/data/person/Address.java]: indentation?

**Comment by OuyangDanwen** [comment]
@li-kai  Generally good effort with some indentation errors.(alignment, when to break to a new line) Also, remember to update Help. 
Please close the PR after reading the comments.

**Comment by li-kai** [comment]
Read comments. I will take note of the indentations from now on.

**Pull request closed without merging**